_pages/algorithm-development.md

```diff
@@ -6,6 +6,7 @@ excerpt: "Basic Guides for Algorithm Developers"
 show_related: false
 redirect_from:
   - /algorithm-development/basic-guides/
+  - /algorithm-development/lang-guides
```

Comment:
Let's add a trailing slash to this one, so that it becomes `/algorithm-development/lang-guides/`. All of the other redirects within the Jekyll app have a trailing slash, and we handle the removal of the trailing slash with Express, so it won't be an issue.